src/features/projectsV2/ProjectListControls/microComponents/ClassificationDropDown.tsx

```diff
@@ -19,12 +18,9 @@ interface ClassificationDropDownProps {
 export const ClassificationDropDown = ({
   selectedClassification,
   setSelectedClassification,
-  isMobile,
   selectedMode,
-  filteredProjects,
 }: ClassificationDropDownProps) => {
```

Comment:
The optional parameters `isMobile` and `selectedMode` were removed here, but are still present in the `ClassificationDropDownProps` type, and are also passed on from `ProjectListControlForMobile`